src/Libraries/CoreNodeModels/ColorRange.cs

```diff
@@ -59,7 +59,7 @@ protected virtual void InitializePorts()
 
         void ColorRange_PropertyChanged(object sender, PropertyChangedEventArgs e)
         {
-            if (e.PropertyName != "IsUpdated")
+            if (e.PropertyName != "CachedValue")
                 return;
```

Comment:
I guess this wasn't quite erroneous, but rather we did not have `CachedValue` to depend on before your change. Yes the frequency can be high, good fix there!